systers_portal/templates/account/password_change.html

```diff
@@ -0,0 +1,21 @@
+{% extends "account/base.html" %}
+{% load url from future %}
+{% load i18n %}
+{% load crispy_forms_tags %}
+{% block head_title %}{% trans "Change Password" %}{% endblock %}
+{% block content %}
+	<div class="mt40"></div>
```

Comment:
I just checked the other account templates, we don't add any margin to the top. It looks ok as it. So you can remove it.